ckanext/dgu/theme/templates/_dgu_jinja_util.html

```diff
@@ -117,6 +117,39 @@
               </button>
             </span>
           </div>
+
+          <div>
+      <span class="dropdown">
+          <a href="#" class="dropdown-button" data-placement="top" data-delay="300" data-toggle="dropdown" title="" data-original-title="Advanced Search" style="color: #666; font-size: 12px; text-decoration: underline; text-align:center; display: block;">
+            Search Tips
+          </a>
+
+          <div class="panel panel-default dropdown-menu" style="width: 500px; margin-left: -1px;">
+            <div class="panel-heading" style="margin-top: -10px;"><strong>Search Tips</strong></div>
+            <div class="panel-body">
+
+                <ul>
+                  <li><span class='search-terms'>"armed forces" expenses</span> - search for exact phrases by putting quote marks around them</li>
+                  <li><span class='search-terms'>primary school -care age NOT "cultural identity"</span> - exclude words or quoted phrases by using a minus sign or NOT</li>
```

Comment:
these are two separate examples. You could put `age NOT "cultural identity"` in a separate span or simply get rid of it.